db-engine/src/main/java/com/splicemachine/db/impl/jdbc/EmbedDatabaseMetaData.java

```diff
@@ -3221,6 +3220,38 @@ private ResultSet doGetIndexInfo(String catalog, String schema, String table,
 		return s.executeQuery();
 	}
 
+	public ResultSet getSchemasInfo() throws SQLException {
+		PreparedStatement s = getPreparedQuery("getSchemasInfo");
+		return s.executeQuery();
+	}
+
+	public ResultSet getTablesForSnaphot(String schemaname, String tablename) throws SQLException {
```

Comment:
Missing `s` in `getTablesForSnaphot`?

Reply:
No missing.

Reply:
Shouldn't it be `getTablesForSnapshot` (with the extra `s`)?